app/services/comments/destroy.rb

```diff
@@ -0,0 +1,27 @@
+module Comments
+  class Destroy
+    def self.call(article_id, comment_id)
+      new(article_id, comment_id).call
+    end
+
+    def call
+      @comment = article.comments.find(comment_id)
```

Comment:
You don’t need to make additional request to find article here
`Comment.find_by(id: id, article_id: article_id)` should be enough.

Plus, not sure what exactly returns `.destroy` method, if it’s an object, you can do
```
def call
  Comment.find_by(id: id, article_id: article_id).destroy
end
```

And, yeah, in Comments namespace you should return `@comment object. Otherwise it will be super confusing.